.github/ISSUE_TEMPLATE/thinout_gitlab_volume_backups.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Thin out GitLab data volume backups
+about: Issue template for quarterly volume cleanup
+title: Thin out GitLab data volume backups
+labels: -,infra,operator,orange
+_start: 2025-04-01T09:00
+_period: 3 months
+---
+In all deployments, keep all backups for the ongoing year, one per month for the previous two years, then one per year.
```

Comment:
```suggestion
In all deployments, keep all backups for the current year, one per month for the previous two years, and one per year for all other years.
```

or consider a bulleted list. Also, we should probably include more details about which one backup to keep, for example the first or last backup per month/year.

- Current Year
  - Keep all
- Previous two years
  - Keep earliest backup from each month
- All other years
  - Keep earliest backup from each year

Reply:
I think it makes more sense for this particular template to keep it in English, the instruction is straight forward.